src/mscorlib/src/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -82,7 +82,7 @@ private unsafe int GetHashCodeOfStringCore(string source, CompareOptions options
 
             fixed (char* pSource = source)
```

Comment:
Does this have the hashcode randomization like the existing CoreCLR Windows version?